### PR TITLE
[READY FOR REVIEW] may + june coworking notes

### PR DESCRIPTION
**Pull request by aleesteele**
### Summary

Notes from may coworking calls ready for review

### CODE REVIEW

**Comment by RichardJActon** [comment]
Hi @aleesteele Is there a convention of someone else who attended the meetings minuted in these notes approving the merge or can anyone do it?

**Comment by aleesteele** [comment]
@RichardJActon Thanks for your question Richard! 

We don't have an established process (yet), but as general practice so far we have had delivery staff team review these PRs to double check any information that should not be archived here. There can of course be multiple reviewers to check for typos, formatting, etc! 👍

**Comment by RichardJActon** [comment]
Thanks @aleesteele good to know - was just checking through any PRs I might be able to give a quick review after book dash.

File: governance/community-calls/20240610-coworking.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Notes 20 May 2024
```
Comment (Arielle-Bennett) [inline governance/community-calls/20240610-coworking.md:1]:
```suggestion
# Notes 10 June 2024
```

File: governance/community-calls/20240610-coworking.md
Comment (Arielle-Bennett) [inline governance/community-calls/20240610-coworking.md:1]: This is a duplicte and/or is missing notes - do we want to add something around them intentionally being left blank if there are no notes to add?

File: governance/community-calls/20240617-coworking.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* Batool, Ale: Scriberia illustrations
+* Anne and Esther Bookdash wash-up
+* Malvika: Reviewing the practitioners hub case study booklet with Ella
+* Cass: Going through some open issues from RCMs etc.
+
+
+## Notes
+
+* Book Dash review: need to discuss with committee about different responsibilities and contigency plans
```
Comment (Arielle-Bennett) [inline governance/community-calls/20240617-coworking.md:29]:
```suggestion
* Book Dash review: need to discuss with committee about different responsibilities and contingency plans
```

File: governance/community-calls/20240520-coworking.md
Comment (Arielle-Bennett) [inline governance/community-calls/20240520-coworking.md:1]: Missing notes? Worth deliberately noting "no notes taken"?

**Review by Arielle-Bennett — APPROVED** [review]
Minor comments and a couple of typoes corrected :)

File: governance/community-calls/20240520-coworking.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Topics/Rooms
+
+* 
+* 
+* 
+
+## Notes
+
+*
```
Comment (aleesteele) [inline governance/community-calls/20240520-coworking.md:23]:
```suggestion
* No notes taken
```

File: governance/community-calls/20240520-coworking.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## Apologies:
+
+* Anne (problem with transit back to UK!)
+* 
+* 
+
+## Topics/Rooms
+
+*
```
Comment (aleesteele) [inline governance/community-calls/20240520-coworking.md:17]:
```suggestion
* No notes taken
```

**Comment by aleesteele** [comment]
Thanks a lot for the review, @Arielle-Bennett! Fixed the missing/duplicate notes, and added in missing notes from July :)

**Pull request merged**